tools/krona/tool_dependencies.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<tool_dependency>
+	<package name="perl" version="5.18.1">
+        <repository name="package_perl_5_18" owner="iuc" prior_installation_required="True" />
+    </package>
+
```

Comment:
But you haven’t defined a dependency here, you are installing it by your own.
You also have not defined a requirement in your tool, afaik.